### ISSUE TITLE
Unit tests: scheduled_tasks install.xml does not have correct permissions

### ISSUE THREAD

**Issue by kristian-94** [catalyst/moodle-local_datacleaner#103]
This recent commit: 
https://github.com/catalyst/moodle-local_datacleaner/commit/155adac527f0ec17d2a6b368a564e3da1bb49d44

Modified file permissions when it shouldn't have. We need to change this back to 644 for the install.xml file and 755 for the db directory.

**Comment by kristian-94** [comment]
@danmarsden  I made a mistake with the last change, and left the file permissions as 777. This looks to be passing travis, when it's green can I merge it?

**Comment by danmarsden** [comment]
Whoops - I should have caught that too!! I've hit that a few times myself too!

**Comment by kristian-94** [comment]
Yes, locally you have to change the permissions to let moodle do the save action, but have to remember to change it back after.

**Comment by jaypha** [comment]
Closing as permissions have since been fixed
